helm-chart/Chart.yaml

```
@@ -14,4 +14,4 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 6.5.1
+version: 6.6.0
```

```suggestion
version: 6.5.1
```
We'll release this with v7